### PR TITLE
unpaper: update to 7.0.0.

### PR DESCRIPTION
**Pull request by zlice**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, **x86_64**

### CODE REVIEW

File: srcpkgs/unpaper/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'unpaper'
 pkgname=unpaper
-version=6.1
-revision=3
-build_style=gnu-configure
+version=7.0.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config meson libxslt"
```
Comment (classabbyamp) [inline srcpkgs/unpaper/template:6]:
```suggestion
hostmakedepends="pkg-config libxslt"
```
done in build_style automatically

**Comment by zlice** [comment]
todo: arm sphinx-build missing?

**Comment by classabbyamp** [comment]
> todo: arm sphinx-build missing?

no, you need it as hostmakedepends

why skip ci?

**Comment by zlice** [comment]
> no, you need it as hostmakedepends

thx. moved that up

**Pull request merged**